src/Http/Routing/src/MapRouteRouteBuilderExtensions.cs

```diff
@@ -154,7 +154,7 @@ public BackCompatInlineConstraintResolver(IInlineConstraintResolver inner, Param
                     return routeConstraint;
                 }
 
-                var parameterPolicy = _parameterPolicyFactory.Create(null, inlineConstraint);
+                var parameterPolicy = _parameterPolicyFactory.Create(null!, inlineConstraint);
```

Comment:
This seems weird. What's going on here?

Reply:
I think I changed the callsite after. This should allow a null.
```suggestion
                var parameterPolicy = _parameterPolicyFactory.Create(null, inlineConstraint);
```